meshroom/ui/qml/GraphEditor/NodeEditor.qml

```diff
@@ -24,6 +24,92 @@ Panel {
     icon: MaterialLabel { text: MaterialIcons.tune }
 
     headerBar: RowLayout {
+        Label {
+            text: {
+                if (node !== null && node.isSubmittedOrRunning()) {
+                    // Some chunks might be submitted but they'll all run eventually
+                    if (node.elapsedTime > 0) { // At least a chunk is done running
+                        return "Running for: " + getTimeStr(node.elapsedTime)
+                    } else {
+                        return (node.chunks.count > 1) ? "First chunk running" : "Node running"
+                    }
+                } else if (node !== null && node.isFinishedOrRunning()) {
+                    /* Either all chunks finished running or the last one is running
+                        * Placed inside an "else if" instead of "else" to avoid entering the functions
+                        * when there is no real use */
+                    return getTimeStr(node.elapsedTime)
+                } else {
+                    return ""
+                }
+            }
+            padding: 2
+            font.italic: true
+            visible: {
+                if (node !== null) {
+                    if ((node.isFinishedOrRunning() || node.isSubmittedOrRunning())) {
+                        return true
+                    }
+                }
+                return false
+            }
+
+            ToolTip.text: {
+                if (node !== null && (node.isFinishedOrRunning() || (node.isSubmittedOrRunning() && node.elapsedTime > 0))) {
+                    var longestChunkTime = getLongestChunkTime(node.chunks)
+                    if (longestChunkTime > 0)
+                        return "(longest chunk: " + getTimeStr(longestChunkTime) + ")"
```

Comment:
No need for ( ) as it's the only message.
```suggestion
                        return "Longest chunk: " + getTimeStr(longestChunkTime)
```

Reply:
Maybe good to display the number of chunks.

Reply:
Done (and number of chunks added)